# Developing Classification Models

## Import packages and modules

In [7]:
import csv
import numpy as np
import pandas as pd
import seaborn as sns
from matplotlib import pyplot as plt

import tensorflow as tf
from tensorflow import keras
from keras import metrics
tf.get_logger().setLevel('INFO')

Using TensorFlow backend.


In [4]:
%run /Users/apassan/Documents/03_Professional/07_GitHubRepo/07_CreditCardFraud/dataset

In [8]:
data_fin = np.asarray(data)

## Model Prep

In [11]:
data['fraud'].value_counts()

0.0    912597
1.0     87403
Name: fraud, dtype: int64

- Based on the above, there are 87,403 fraud transactions, and 912,507 non-fraud transactions in the dataset. This means that only \~9.5% of the dataset has fraudulent transactions.